### PR TITLE
updated namereg.sol contract

### PR DESCRIPTION
**Pull request by debris**

### CODE REVIEW

File: namereg/namereg.sol [hunk 1]
```diff
@@ -4,25 +4,23 @@
...
 
-#require service, owned
+import "service";
+import "owned";
 contract NameReg is service(1), owned, NameRegister {
   	event AddressRegistered(address indexed account);
   	event AddressDeregistered(address indexed account);
 
 	function NameReg() {
-		toName[Config()] = "Config";
```
Comment (chriseth) [inline namereg/namereg.sol]: The config contract is not there anymore?

Comment (debris) [inline namereg/namereg.sol]: I don't know, I just wanted to be able to compile this contract. It was failing with following error:
```
namereg.sol:18:16: Type error: More than one argument for explicit type conversion.
        toName[Config()] = "Config";
               ^------^
```

Comment (chriseth) [inline namereg/namereg.sol]: Ah, ok, then it never worked before ;-)

**Comment by gavofyork** [comment]
NameReg is deprecated - use GlobalRegistrar, OwnedRegistrar or Registrar instead.

**Pull request closed without merging**